deploy/docs/v1_conf_examples.md

```diff
@@ -0,0 +1,130 @@
+        </buffer>
+      </match>
+```
+
+### Adding Fluentd Filter plugin
+
+You can add any of the fluentd filter plugins in the `extraFilterPluginConf` section to filter data as per your needs.
+
+The below example uses the `grep` filter to match any event that satisfies the following conditions:
+ - The value of the "message" field contains "cool"
+ - The value of the "message" field does NOT contain "uncool"
+
+```yaml
+fluentd:
+  logs:
+    containers:
+      extraFilterPluginConf: |-
+        <filter containers.**>
+          @type grep
+          <regexp>
+            key message
+            pattern /cool/
+          </regexp>
+          <exclude>
+            key message
+            pattern /uncool/
+          </exclude> 
+        </filter>
+```
+Reference documentation: [Fluentd Filter Plugin](#https://docs.fluentd.org/filter)
+
+## Override Fluentd Output plugin to forward data to Sumo as well as S3
+The example below shows how you can override the entire output section for the container logs pipeline.
+
+```yaml
+fluentd:
+  logs:
+    containers:
+      overrideOutputConf: |-
+        <match containers.**>
+          @type copy
+          <store>
+            @type sumologic
+            @id sumologic.endpoint.logs
+            @include logs.output.conf
+            <buffer>
+              @type file
+              path /fluentd/buffer/logs.containers
+              @include buffer.output.conf
+            </buffer>
+          </store>
+          <store>
+            @type s3
+            aws_key_id YOUR_AWS_KEY_ID
+            aws_sec_key YOUR_AWS_SECRET_KEY
+            s3_bucket YOUR_S3_BUCKET_NAME
+            s3_region YOUR_S3_BUCKET_REGION
+            path logs/
+            # if you want to use ${tag} or %Y/%m/%d/ like syntax in path / s3_object_key_format,
+            # need to specify tag for ${tag} and time for %Y/%m/%d in <buffer> argument.
+            <buffer tag,time>
+              @type file
+              path /fluentd/buffer/logs.containers.s3
+              timekey 3600 # 1 hour partition
+              timekey_wait 10m
+              timekey_use_utc true # use utc
+              chunk_limit_size 256m
+            </buffer>
+          </store>
+        </match>
+```
+
+### Additional Buffer/Flush/Retry Config parameters for Sumologic Output Plugin
+
+The following confgig parameters are set by default and their values can be set by changing the respective cofig in `values.yaml`. 
+
+```bash
+compress gzip
+flush_interval {{ .Values.fluentd.buffer.flushInterval | quote }}
+flush_thread_count {{ .Values.fluentd.buffer.numThreads | quote }}
+chunk_limit_size {{ .Values.fluentd.buffer.chunkLimitSize | quote }}
+total_limit_size {{ .Values.fluentd.buffer.totalLimitSize | quote }}
+queued_chunks_limit_size {{ .Values.fluentd.buffer.queueChunkLimitSize | quote }}
+overflow_action drop_oldest_chunk
+```
+If you want to add any new Buffer/Flush/Retry Config, you can add them in the `extraConf` section.
```

Comment:
To help distinguish between the above configs which are controlled with parameters in `values.yaml`, and the ability to add other configs through this new `extraConf` section, I think we could tweak this line to something along the lines of
```
However, if you wish to add any additional Buffer/Flush/Retry configs, you can do so in the new `extraConf` section.
```